app/utils/encryption.py

```
@@ -38,12 +53,59 @@ def encrypt(cls, plaintext: str) -> str:
+    async def decrypt_for_user(cls, encrypted_text: str, user_id: int) -> str:
+        """
+        Decrypts data for a specific user using their stored salt.
+
+        Args:
+            encrypted_text: Data to decrypt
+            user_id: User's ID
+
+        Returns:
+            Decrypted string
+        """
+        from app.models import User
```

Pylint (3.3.7)

[convention] Import outside toplevel (app.models.User)

(C0415)